operator/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Main
 
+- [11288](https://github.com/grafana/loki/pull/11288) **periklis**: Add custom objstore SA for ruler component
```

Comment:
I'm honest: I did not know what you meant with "SA" in this context. I have changed the title of the issue to reflect my understanding, please update this line, if you agree:

```suggestion
- [11288](https://github.com/grafana/loki/pull/11288) **periklis**: Fix custom CA for object-store in ruler component
```